R/noms_date_range_pull.R

```diff
@@ -0,0 +1,51 @@
+library(googleAnalyticsR)
+library(googleAuthR)
+library(arrow)
+library(tidyr)
+library(lubridate)
+library(plyr)
+library(dplyr)
+
+yesterday <- seq(from=as.Date('2019-10-01'), to=as.Date('2020-09-30'), by='days')
+
+get_nwis_site_page_views <- function(date_range = c('2020-01-01', '2020-02-01')) {
+  gar_auth_service('~/.vizlab/VIZLAB-a48f4107248c.json')
+  nwis_web_view <- '49785472'
+  
+  #filter 
+  #get all pagePaths with numeric data between 8 and 15 digits long in it:
+  filter_8_15_digit_site <- dim_filter("pagePath", "REGEXP", expressions = "[0-9]{8,15}")
+  #########
+  
+  ###########
+  filter_clause_site_page <- filter_clause_ga4(list(filter_8_15_digit_site),
+                                               operator = "AND")
+  
+  flow_stage_or_q <- google_analytics(nwis_web_view, date_range = date_range,
```

Comment:
```suggestion
site_number_page_views <- google_analytics(nwis_web_view, date_range = date_range,
```
That variable name is a bit misleading